staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/fuzzer/fuzzer.go

```diff
@@ -66,6 +66,7 @@ func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
 					Strategy: apiextensions.NoneConverter,
 				}
 			}
+			obj.Conversion.ConversionReviewVersions = []string{"v1beta1"}
```

Comment:
only if len == 0?

Reply:
Hmm, is the point of this function to make the object valid? any value other than []string{"v1beta1"} is not valid.

Reply:
fuzzing doesn't care about validation, just about round-trippability. len(0) doesn't round-trip because empty lists get defaulted